code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -1001,6 +1001,123 @@
 	..()
 	M.heal_organ_damage(0.5 * removed, 0)
 
+/datum/reagent/drink/coffee/espresso
+	name = "Espresso"
+	id = "espresso"
+	description = "A strong coffee made by passing nearly boiling water through coffee seeds at high pressure."
+	color = "#664300" // rgb: 102, 67, 0
+	adj_temp = 5
+	taste_description = "bitter coffee"
+
+	glass_icon_state = "hot_coffee"
+	glass_name = "shot of espresso"
+	glass_desc = "A strong coffee made by passing nearly boiling water through coffee seeds at high pressure."
+	glass_center_of_mass = list("x"=15, "y"=9)
+
+/datum/reagent/drink/coffee/freddo_espresso
+	name = "Freddo espresso"
+	id = "freddo_espresso"
+	description = "Espresso with ice cubes poured over ice."
```

Comment:
That seems a bit off.
How about "Espresso poured over ice cubes .

Reply:
https://en.wikipedia.org/wiki/Iced_coffee#Greece
"Other popular iced coffee beverages in Greece include Freddo Cappuccino [...] and Freddo Espresso, which are made with a double shot of espresso coffee mixed in a mixer _with a few ice cubes and poured over ice_"

Reply:
Wikipedia can be written by anyone.